node-fs-writefile/cp.js

```diff
@@ -0,0 +1,13 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+async function cp() {
+  try {
+    const contents = await readFile(process.argv[2], 'utf8');
+    const copyFile = String(process.argv[3]);
+    await writeFile(copyFile, `${contents}\n`);
+  } catch (error) {
+    console.log(`Error: ${error}`);
```

Comment:
This doesn't properly log the error. Instead, it converts `error` to a string, which since it's an object will be `[object Object]`. You want to have `error` be an argument to console.log:
`console.log('Error:', error);`

Reply:
Changed from obj. literal to console.log error